wagtail/wagtailcore/blocks/static_block.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.utils.safestring import mark_safe
+
+from .base import Block
+
+__all__ = ['StaticBlock']
+
+
+class StaticBlock(Block):
+    """
+    A block that just 'exists' and has no fields.
+
+    It's an abstract model, so it has to be subclassed.
+    Subclasses should define self.admin_html in order to
+    display something in the admin even though the block
+    has no fields.
+    """
+    def render_form(self, value, prefix='', errors=None):
+        admin_html = getattr(self, 'admin_html', '')
```

Comment:
Providing a useful default would help developers. An empty string will never be appropriate, and could cause developers to think something is broken. A useful default will suite some simple cases, and will not look broken even if the developer eventually wants to change the message. Assuming that the `admin_html` attribute is moved to `Meta` as I suggested above, something like this would work:

```python
def render_form(self, value, prefix='', errors=None):
    if self.meta.admin_html is None:
        return _('{label}: This block has no options').format(label=self.label)
    return self.meta.admin_html
```